content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -193,116 +190,111 @@ This procedure assumes the Windows administrator user is QE. Substitute your sys
 
-Enter the XML metadata  file from Google Admin in the  SAML Identity Provider Metadata XML Upload field. After uploading, click Save.
+Enter the XML metadata file from Google Admin into the **SAML Identity Provider Metadata XML Upload** field, then click **Save**.
 
-Click Start the SAML service checkbox and click Save to start the service.
+Click the **Start the SAML service** checkbox, then click **Save** to start the service.
 
 Log out of TrueCommand.
 
 ## Configuring a Google Admin SAML App
 
-1. Open Google Admin and go to Apps > Web and mobile apps
+1. Open Google Admin and go to **Apps > Web and mobile apps**
 
    ![GASelectWebandMobileApps](/images/SAML/GASelectWebandMobileApps.jpg "GA Select Web and Mobile Apps")
     
-2. Click Add App and select Add custom SAML app.The Add custom SAML app screen App details screen displays. 
+2. Click **Add App**, then select **Add custom SAML app**.
 
    ![GAClickAddApp](/images/SAML/GAClickAddApp.jpg "GAClickAddApp")
     
 3. Configure the SAML app screen settings. 
 
    ![GAAddCustomSAMLAppDetailsCropped](/images/SAML/GAAddCustomSAMLAppDetailsCropped.jpg "GA Add Custom SAML App Details Cropped")
     
-   a. Type any name you want to use in the App Name field. This example uses tcsaml.
-   b. Upload any picture or avatar you want to use into the App icon area to identify the app in your google admin account.
-   c. Click **CONTINUE.** The Google Identity Provider details screen displays. 
+   a. Type any name you want to use in the **App Name** field. This example uses *tcsaml*.
+   b. Upload any picture or avatar you want to use into the **App icon** area to identify the app in your Google Admin account.
+   c. Click **CONTINUE.** to view the **Google Identity Provider** screen. 
 
-4. Click **CONTINUE**. The Service provider details screen displays.
+4. Click **CONTINUE** to view the **Service Provider** details screen.
 
 5. Configure the service provider details. 
     
    ![GAAddCustomSAMLAppAddServiceProviderDetailsCropped](/images/SAML/GAAddCustomSAMLAppAddServiceProviderDetailsCropped.jpg "GA Add Custom SAML App Add Service Provider Details Cropped")
     
-   a. Type or copy/paste the TrueCommand login URL http://IP:PORT/saml/acs in the into the ACS Url field. Where IP:PORT is your TrueCommand system IP and port address. 
-   b. Type any name you want to use into the Entity ID field. For example, truecommand-saml.
-   c. Type the https://IP:PORT/saml/helloURL into the Start URL field. Where IP:PORT is your TrueCommand system IP and port address.
-   d. Set Name ID format to PERSISTENT.
+   a. Type or copy/paste the TrueCommand login URL *http://IP:PORT/saml/acs* into the **ACS Url** field. *IP:PORT* is your TrueCommand system IP and port address. 
+   b. Type any name you want into the **Entity ID** field (ex. truecommand-saml).
+   c. Type the *https://IP:PORT/saml/helloURL* into the **Start URL** field. *IP:PORT* is your TrueCommand system IP and port address. 
```

Comment:
same issue with URL and variable. suggest:

   c. Type the **https://*IP:PORT*/saml/helloURL** into the **Start URL** field. *IP:PORT* is your TrueCommand system IP and port address.

Reply:
Fixed across the board!